# pd.Series.mean()

Siehe auch: [Arithmetisches Mittel](../Formeln/arithmetic_mean.ipynb)

`pd.Series.mean()` ist eine Methode in Pandas, die den Durchschnitt (arithmetisches Mittel) aller numerischen Werte in einer Pandas Series berechnet. Eine Series ist eine einspaltige Datenstruktur in Pandas, die eine Liste von Werten enthält. 

### Funktionsweise

Die Methode `mean()` berechnet den Durchschnitt, indem sie die Summe aller numerischen Werte in der Series durch die Anzahl dieser Werte teilt. Falls die Series `NaN`-Werte (fehlende Werte) enthält, ignoriert `mean()` diese standardmäßig.

### Syntax

```python
Series.mean(skipna=True)
```

- **`skipna`**: Ein optionaler Parameter. Wenn `True` (Standard), werden fehlende Werte (`NaN`) ignoriert. Bei `False` würde ein `NaN` in der Series das Ergebnis ebenfalls zu `NaN` machen.

### Beispiel

Angenommen, wir haben eine Serie von Zahlen, die die Umsätze eines Geschäfts in einer Woche darstellen:

In [2]:
import pandas as pd

umsatz = pd.Series([100, 200, 300, None, 500])

# Berechnung des Durchschnitts
durchschnitt = umsatz.mean()
print("Durchschnittlicher Umsatz:", durchschnitt)

Durchschnittlicher Umsatz: 275.0


### Erklärung

- Die Methode ignoriert den `None`-Wert (entspricht `NaN`) und berechnet den Durchschnitt der restlichen Werte `(100 + 200 + 300 + 500) / 4 = 275`.
- Wenn `skipna=False` gesetzt ist, würde `durchschnitt` zu `NaN` werden, da die Serie fehlende Werte enthält.

### Anwendung

`pd.Series.mean()` ist hilfreich, um schnell den Durchschnitt einer Spalte in einem DataFrame zu berechnen oder allgemeine Zusammenfassungen von numerischen Daten zu erhalten.